src/utils.ts

```diff
@@ -1,8 +1,11 @@
+import { clonedMathFloor, clonedMathRandom } from "./math";
```

Comment:
Let's have the `originalMathFloor` and `originalMathRandom` declared and exported from utils so no file would need to import from a sibling file other than from utils